config/testing/vpa_namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-vertical-pod-autoscaler
```

Comment:
I'd rather not a create this file just to add a namespace to the test environment. Can we just create it  directly somehow in the setup, or is that not possible?